src/components/spatialdisplay/SpatialDisplayComponent.vue

```diff
@@ -28,6 +28,20 @@
       :locationId="props.locationId"
       @changeLocationId="onLocationChange"
     />
+    <InformationPanel
+      :layerTitle="props.layerCapabilities?.title"
+      :currentTime="currentTime"
+      :forecastTime="forecastTime"
+      :styles="styles"
+      :completelyMissing="
+        props.layerCapabilities?.completelyMissing ? true : false
```

Comment:
This will probably be wrong now if completelyMissing is defined but set to false.
Could be `?? false`